packages/components/src/Card/CardClickable.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode, createRef } from "react";
+
+interface ClickableCardProps {
+  onClick(event: React.MouseEvent<HTMLElement>): void;
+  className: string;
+  readonly children: ReactNode | ReactNode[];
+}
+
+const ENTER_KEY = "Enter";
+const SPACEBAR_KEY = " ";
+
+/**
+ * This is only intended to be used in the Card component.
+ * Please use `<Card onClick={onClick} />` component instead.
+ */
+export function CardClickable({
```

Comment:
Couldn't find an example for a nested components naming so I went with the whole `<ParentChild />` naming style since this component should only be used by the `Card`.

Reply:
👍🏻  Not sure what Atlantis convention is but if it was up to Shopify this would have been called ClickCards. Weird I know. I like plain old [Domain]|[Context]|Component|[Type] approach

Reply:
We kind of have one but not officially(?). 

![image](https://user-images.githubusercontent.com/15986172/131405944-894311cf-167e-4e42-9f07-d94d441d36e2.png)

I agree tho, `ClickableCard` actually makes sense. I just went with it being under the `Card` umbrella.

Reply:
💯